examples/samples/README.md

```diff
@@ -2,42 +2,40 @@
 
 ## Building the library
 - Clone the repo and cd to the repo directory
-```
+```bash
 git clone https://github.com/azure/azure-service-bus-node.git
 cd azure-service-bus-node
 ```
 - Install typescript, ts-node globally (optional, but very useful)
-```
+```bash
 npm i -g typescript
 npm i -g ts-node
 ```
 - NPM install from the root of the package
-```
+```bash
 npm i
 ```
 - Build the project
-```
+```bash
 npm run build
 ```
 
 ## Before executing a sample
-- Go to the Azure Portal(https://portal.azure.com)
-- Here are the docs which would help you create a service bus resource in the portal : https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-nodejs-how-to-use-queues
-```
-Go to Dashboard > Service Bus > <your-servicebus-namespace> 
-```
+- Go to the [Azure Portal](https://portal.azure.com)
+- Here are the docs which would help you create a service bus resource in the portal : [ServiceBus - NodeJS DOCS](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-nodejs-how-to-use-queues)
+- In the portal, go to **Dashboard > Service Bus > _your-servicebus-namespace_**
 - If you'd like to work with Queues, find the "Queues" tab right under "Entities" at <your-servicebus-namespace> and create a Queue.
```

Comment:
@ShivangiReja Once this PR is checked in, can you add a note here about working with Topic and Subscription, similar to what is there here for Queues?

Reply:
Okay.